config/env.production

```diff
@@ -112,6 +112,30 @@ POSTS_URL=https://api.telescope.cdot.systems/v1/posts
 MOCK_REDIS=
 
 
+################################################################################
+# User Service
+################################################################################
+
+# Firebase's private api key. Used to authenticate telescope with firebase
+# See https://firebase.google.com/docs/admin/setup#initialize-sdk to obtain the key
+FIREBASE_TYPE=
+FIREBASE_PROJECT_ID=
+FIREBASE_PRIVATE_KEY_ID=
+FIREBASE_PRIVATE_KEY=
+FIREBASE_CLIENT_EMAIL=
+FIREBASE_CLIENT_ID=
+FIREBASE_AUTH_URI=
+FIREBASE_TOKEN_URI=
+FIREBASE_AUTH_PROVIDER_X509_CERT_URL=
+FIREBASE_CLIENT_X509_CERT_URL=
+
+# User Service Port (default is 6666)
+USER_PORT=6666
```

Comment:
Can you also update `USER_*` to `USERS_*` (plural) in all these files, as in, "the Users Service."